pipeline_dp/accumulator.py

```diff
@@ -28,18 +29,28 @@ def create_accumulator_params(
     budget_accountant: budget_accounting.BudgetAccountant
 ) -> typing.List[AccumulatorParams]:
     accumulator_params = []
-    budget = budget_accountant.request_budget(
-        aggregation_params.noise_kind.convert_to_mechanism_type())
     if pipeline_dp.Metrics.COUNT in aggregation_params.metrics:
-        count_params = CountParams(budget, aggregation_params)
+        budget_count = budget_accountant.request_budget(
+            aggregation_params.noise_kind.convert_to_mechanism_type())
```

Comment:
Nit: in order to improve readability please extract `mechanism_type` to a local variable